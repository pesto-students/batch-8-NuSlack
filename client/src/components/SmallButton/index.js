import React from 'react';
import { Button } from 'antd';

const SmallButton = props => <Button size="default" {...props} />;

export default SmallButton;
