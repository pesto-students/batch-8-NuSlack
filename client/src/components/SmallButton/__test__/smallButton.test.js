import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SmallButton from '../index';

describe('<SmallButton />', () => {
    it('should match the snapshot', () => {
        const tree = shallow(<SmallButton />);
        expect(toJson(tree)).toMatchSnapshot();
    });
});
