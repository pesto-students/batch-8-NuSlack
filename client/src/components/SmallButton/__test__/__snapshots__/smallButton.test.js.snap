// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmallButton /> shallow should match the snapshot 1`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
  size="default"
/>
`;

exports[`<SmallButton /> shallow should match the snapshot 2`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
  size="default"
/>
`;
